packages/mdc-theme/_functions.scss

```diff
@@ -16,6 +16,11 @@
 
 @import "./constants";
 
+// Used by the functions below to shift a color's lightness by approximately
```

Comment:
Would "luminance" or "shade" possibly make more sense here than "lightness"?

Reply:
I'm not sure which term is technically correct (the best kind of correct!).

[libsass's implementation of `lighten` and `darken`](https://github.com/sass/libsass/blob/0dea5f2d6b6b13e39a0d66b5ee45e29ed362c029/src/functions.cpp#L566) converts the input color to HSL, increases the 'L', and returns the result.

According to [Wikipedia's article on HSL](https://en.wikipedia.org/wiki/HSL_and_HSV), the 'L' stands for "luminance" _or sometimes_ "lightness", which the [article later defines as being different things](https://en.wikipedia.org/wiki/HSL_and_HSV#Formal_derivation).

In addition, we don't use Sass's built-in `lightness` function - instead, we implement our own `mdc-theme-luminance` function based on a [W3 spec](https://www.w3.org/TR/WCAG20-TECHS/G17.html#G17-tests), which makes me think that there are subtly different and competing usages of the words "lightness" and "luminance" floating around the Web and in our codebase.

In short, I have no idea which word to use 😝